config/sync/core.extension.yml

```diff
@@ -43,6 +43,7 @@ module:
   flag: 0
   flag_lists: 0
   gcontent_moderation: 0
+  ggroup: 0
```

Comment:
Ggroup is only used a few hundred times, is not under security upgrades and the devs don't seem to be too active. Did you evaluate the code and the support situation?

Reply:
Did a quick code review previously and code seems to be within best practices and well done. There's quite a lot to be done feature wise but that would be the case either way.

Compared to subgroup-module the usage statistics (358 vs 361) are essentially the same. 
So is the maintenance status (subgroup last update may 2021 and ggroup february 2021).

Basically the only difference between the two when it comes to maintenance is that subgroup has stable release and ggroup does not.